libvirt.go

```diff
@@ -304,6 +316,10 @@ func (d *Driver) Start() error {
 		return err
 	}
 
+	if d.Network == "" {
+		return nil
+	}
+
```

Comment:
Hmm this means we have no way of knowing if the VM successfully booted or not? :-/

Reply:
Yes, sadly. The current method is not particularly better: just polling the IP. We could probably scan the console and wait for the login pompt.